storage/series_cursor_test.go

```diff
@@ -0,0 +1,38 @@
+package storage
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/tsdb"
+)
+
+func Test_NewSeriesCursor_UnexpectedOrg(t *testing.T) {
+	makeKey := func(org, bucket uint64) []byte {
+		name := tsdb.EncodeName(influxdb.ID(org), influxdb.ID(bucket))
+		return tsdb.AppendSeriesKey(nil, name[:], nil)
+	}
+
+	nm := tsdb.EncodeName(0x0f0f, 0xb0b0)
+	cur := &seriesCursor{
+		keys: [][]byte{
+			makeKey(0x0f0f, 0xb0b0),
+			makeKey(0xffff, 0xb0b0),
+		},
+		orgID: nm[:8],
```

Comment:
Can we change `[:8]` to `[:influxdb.OrgIDLength]` or something? I just want to make it simpler to change the id scheme if we need to.

Reply:
Yeah, sounds good; I'll use that constant in the `newSeriesCursor` to validate too